### DISCUSSION TITLE
WPNotif: WordPress SMS & WhatsApp Message Notifications

### DISCUSSION THREAD

**Discussion by logeshchenniappan** [inspireui/support#5390] [New Feature/Ideas Requests]
Wpnotif lets you send notifications using SMS or WhatsApp to your customers from your WordPress site. The plugin is also fully compatible with WooCommerce and you can setup automatic notification triggers for your customers whenever they create a new order or whenever the status of that order is changed. Site admins including, store owners, editors etc can also setup messages or WhatsApp notifications for themselves.

**Comment by Yan-Aleman** [comment]
> Wpnotif lets you send notifications using SMS or WhatsApp to your customers from your WordPress site. The plugin is also fully compatible with WooCommerce and you can setup automatic notification triggers for your customers whenever they create a new order or whenever the status of that order is changed. Site admins including, store owners, editors etc can also setup messages or WhatsApp notifications for themselves.

Does he not send sms if the purchase was made through the app? (webview checkout page)
